README.md

```diff
@@ -14,9 +19,9 @@
 
 👤 **Toma John Musa**
 
-- GitHub: [@githubhandle](https://github.com/Kamba56)
-- Twitter: [@twitterhandle](https://twitter.com/Kamba_TJ)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/toma-john-47092622b)
+- GitHub: [@Kamba56](https://github.com/Kamba56)
+- Twitter: [@Kamba_TJ](https://twitter.com/Kamba_TJ)
+- LinkedIn: [Toma](https://linkedin.com/in/toma-john-47092622b)
 
 ## Show your support
 
```

Comment:
[OPTIONAL] The MIT link is broken. Kindly fix it. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

Reply:
@medaminedev66 Please I don't know what link you are talking about, i checked and everything works fine.